sql/pgwire_test.go

```diff
@@ -168,12 +168,12 @@ func TestPGPrepareFail(t *testing.T) {
 	testFailures := map[string]string{
 		"SELECT $1 = $1":                             "pq: unsupported comparison operator: <valarg> = <valarg>",
 		"SELECT $1 > 0 AND NOT $1":                   "pq: incompatible NOT argument type: int",
-		"SELECT $1":                                  "pq: unsupported result type: valarg",
+		"SELECT $1":                                  "pq: could not determine data type of parameter $1",
 		"SELECT $1 + $1":                             "pq: unsupported binary operator: <valarg> + <valarg>",
```

Comment:
I mentioned this in another PR, but it would be nice to replace all the uses of `valarg` in error messages with something more user friendly, like `parameter`.